var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -255,18 +264,28 @@ def patch(self):
                         '-I{0}'.format(spec['zlib'].prefix.include),
                         'gcc/Makefile.in')
 
+    # https://gcc.gnu.org/install/configure.html
     def configure_args(self):
         spec = self.spec
 
         # Generic options to compile GCC
         options = [
+            # Distributor options
+            '--with-pkgversion=Spack GCC',
+            '--with-bugurl=https://github.com/spack/spack/issues',
+            # Xcode 10 dropped 32-bit support
             '--disable-multilib',
+            # Don't bother with full 3-stage bootstrapping process
+            # Builds much faster, but doesn't test build as well
+            # See https://gcc.gnu.org/install/build.html
+            '--disable-bootstrap',
             '--enable-languages={0}'.format(
                 ','.join(spec.variants['languages'].value)),
+            # Drop gettext and libiconv dependencies
+            '--disable-nls',
+            '--with-libiconv-prefix={0}'.format(spec['libiconv'].prefix),
             '--with-mpfr={0}'.format(spec['mpfr'].prefix),
             '--with-gmp={0}'.format(spec['gmp'].prefix),
-            '--enable-lto',
-            '--with-quad'
```

Comment:
This option doesn't appear to exist...

Reply:
This is an option specifically for gfortran in earlier versions. You can make it version dependent if needed. We only build the gcc package on macOS to get fortran.

Reply:
I looked at the Homebrew gcc@4.9 formula and this option doesn't exist there. This must be a vestigial option from before the quad math library was always built.